src/resizable-card-layout.scss

```diff
@@ -0,0 +1,135 @@
+@import "./new-settings";
+@import "./mixins";
+@import "./icon";
+
+$block: #{$fd-namespace}-resizable-card-layout;
+
+.#{$block} {
+  $fd-resizable-card-min-width: 20rem;
+  $fd-resize-handle-width: 0.0625rem;
+  $fd-resize-icon-padding: 0.0625rem;
+  $fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor);
+
+  // reduced by 0.5rem as card already have 0.5rem padding
+  $fd-resizable-card-padding-x: (
+    sm: 0,
+    md: 0.5rem,
+    lg: 2.5rem,
+  );
+
+  @include fd-reset();
+
+  padding-left: 0.5rem;
+  padding-right: 0.5rem;
+
+  &__item {
+    @include fd-reset();
+    @include fd-flex();
+
+    min-width: $fd-resizable-card-min-width;
+    margin: 0.5rem;
+    position: relative;
+  }
+
+  &__card {
+    @include fd-reset();
+
+    order: 1;
+    flex: 1;
+    position: relative;
+    // subtract width taken by vertical handler and horizontal handler
+    height: calc(100% - #{$fd-resize-handle-width});
+  }
+
+  &__icon-wrapper {
+    @include fd-reset();
+
+    width: 1rem;
+    height: 1rem;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    z-index: 2;
+    padding: 0 $fd-resize-icon-padding $fd-resize-icon-padding 0;
+
+    @include fd-rtl() {
+      right: auto;
+      left: 0;
+      padding: 0 0 $fd-resize-icon-padding $fd-resize-icon-padding;
+    }
+  }
+
+  &__icon {
+    @include fd-reset();
+    @include fd-icon("resize-corner");
+
+    font-family: SAP-icons;
+    font-size: 1rem;
+    line-height: 1rem;
+    cursor: nwse-resize;
+    position: absolute;
+    bottom: 0;
+    right: $fd-resize-icon-padding;
+    color: var(--sapButton_IconColor);
+
+    @include fd-rtl() {
+      transform: rotate(90deg);
+      position: relative;
+      cursor: nesw-resize;
+      padding: 0 $fd-resize-icon-padding $fd-resize-icon-padding 0.5rem;
+      // padding-left is needed after rotation
+    }
+
+    &--vertical {
+      cursor: ns-resize;
+
+      @include fd-rtl() {
+        cursor: ns-resize;
+      }
```

Comment:
Is it still needed?

Reply:
at line 78. cursor changing from north-west to north-east in rtl mode.
we are overriding this cursor in --vertical/--horizontal. same has to be done for rtl as well, otherwise it is taking cursor: nesw-resize; in rtl mode for --vertical/--horizontal also.

Reply:
@JKMarkowski please see my comment.